### PR TITLE
2.8.0 containers reduction

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -375,6 +408,107 @@ patch__compose__remove_invoice_send_email() {
...
+        return 1
+    fi
+}
+
+patch__compose__add_udp_to_web_app() {
+    if ! (is_updating_to_version "${UPDATING_TO}" "2007000" 0 1); then
+        return 1
+    fi
+
+    if ! cat -vt "${UCRM_PATH}/docker-compose.yml" | grep -Eq "      - 2055:2055/udp";
```
Comment (keksa) [inline update.sh]: this port can be manually customized by user, customization should be kept and only moved to web_app container

Comment (enumag) [reply update.sh]: I have no idea how to make that work.

File: update.sh [hunk 2]
```diff
@@ -20,6 +20,8 @@ if [[ "${UCRM_PATH}" = "" ]]; then
     UCRM_PATH="."
 fi
 
+UDP_PORT=$(cat -vt "${UCRM_PATH}/docker-compose.yml" | grep --color=never ":2055/udp" || echo "      - 2055:2055/udp")
```
Comment (keksa) [inline update.sh]: limit grep to single line

File: update.sh [hunk 3]
```diff
@@ -375,6 +410,107 @@ patch__compose__remove_invoice_send_email() {
...
+
+patch__compose__add_udp_to_web_app() {
+    if ! (is_updating_to_version "${UPDATING_TO}" "2007000" 0 1); then
+        return 1
+    fi
+
+    if ! cat -vt "${UCRM_PATH}/docker-compose.yml" | grep -Eq ":2055/udp";
+    then
+        echo "Your docker-compose does not contain configuration for port 2055. Trying to add."
+        sed -i -e "s/      - 8443:443/&\n${UDP_PORT/\//\\\/}/g" "${UCRM_PATH}/docker-compose.yml"
```
Comment (keksa) [inline update.sh]: `s|      - 8443:443|&\n${UDP_PORT}|g`

also this will not work if the HTTPS port is different than 8443 (most cases)

**Pull request merged**